hoot-services/src/main/java/hoot/services/controllers/grail/WaitOverpassUpdate.java

```diff
@@ -150,8 +150,8 @@ public CommandResult execute() {
                 DbUtils.tagTimeoutTask(jobId);
                 String msg = "Overpass sync wait time exceeded.";
                 throw new WebApplicationException(new NotFoundException(), Response.status(Response.Status.BAD_REQUEST).entity(msg).build());
-            } else if (this.params.getApplyTags()) {
-                DbUtils.removeTimeoutTag(jobId);
+            } else if (this.params.getTaskInfo() != null) {
+                DbUtils.removeTimeoutTag(this.params.getTaskInfo());
```

Comment:
So this removes the timeout tag from any job status row for the same task (project # and cell #)?  Wouldn't the upload job (using job id) be the only job status row needed this?  Or is it possible to have multiple upload jobs for the same task?

Reply:
now that I added the run button back a user can hypothetically run a task, it times out and gets tagged, run it again, times out and tagged, run resume and it passes, we would have to remove the tag for 2 rows